kernel/cPrimitives.mli

```diff
@@ -70,32 +80,49 @@ val hash : t -> int
 val to_string : t -> string
 
 val arity : t -> int
+(** Including parameters *)
+
+val nparams : t -> int
 
 val kind : t -> args_red
+(** Includes parameters. TODO? use option instead of having Kparam *)
```

Comment:
@SkySkimmer can you elaborate on what you wanted here?

Reply:
I forgot, or perhaps my understanding of kparam shifted so that I can't come up with the same idea again. Since it's TODO-question-mark we can just drop it without worries IMO